src/Client.php

```diff
@@ -359,6 +359,18 @@ public function send($method, $payload, $path, $params = [], array $headers = []
 
         if ($response->getStatusCode() === 422) {
             $content = json_decode($response->getBody()->getContents(), true);
+
+            if (isset($content['invalidFields']) && is_array($content['invalidFields'])) {
+                foreach ($content['invalidFields'] as $field => &$errors) {
```

Comment:
This might be moved to `DataValidationException`

Reply:
btw, `DataValidationException` may acceot whole payload into constructor and use it

Reply:
for example `DataValidationException` may have 2 methods, one return array or errors as is, with fields and other error fields, and other only list of messages